mne/preprocessing/tests/test_ica.py

```diff
@@ -489,12 +489,19 @@ def f(x, y):
 
         ica.labels_ = None
         idx, scores = ica.find_bads_ecg(epochs, method='ctps')
+
         assert_equal(len(scores), ica.n_components_)
         assert_raises(ValueError, ica.find_bads_ecg, epochs.average(),
                       method='ctps')
         assert_raises(ValueError, ica.find_bads_ecg, raw,
                       method='crazy-coupling')
 
+        # check passing a ch_name to find_bads_ecg
+        ica.find_bads_ecg(epochs, raw.ch_names[0], method='ctps')
+
+        idx, scores = ica.find_bads_eog(raw)
+        assert_equal(len(scores), ica.n_components_)
```

Comment:
can you at least check that the scores are different when you pass a ch_name ? maybe also the best score should be less if you pass a bad ecg channel?